libs/util/src/lib/generate-avatar-link.ts

```diff
@@ -1,16 +1,23 @@
-export const generateAvatarLink = async (
-  playerName: string,
-): Promise<string> => {
+import { TextEncoder } from 'util';
+global.TextEncoder = TextEncoder;
```

Comment:
What does this line do?

Reply:
Removed. Global text encoder initialization was added to prevent an error while serving the react app. But the error cannot be reproduced anymore.

Reply:
1e9960e3f5635b25908d286bf437999de16ad720